README.md

```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01 
+Yixi Chen
+
+### Yixi's Bio
+
+Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:
+1. Granada, Spain
+2. Kyoto, Japan
+3. Jeruselum, Isareal  
```

Comment:
Typo in Israel. RStudio actually has a built-in spell checker you can use to proof-read Markdown documents